share/spice/sales_tax_holiday/sales_tax_holiday.js

```diff
@@ -0,0 +1,70 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_sales_tax_holiday = function(api_result){
+        // Error checking
+        var errorMessage = "UNKNOWN ERROR";
+        var isError = false;
+        if (!api_result 
+         || (typeof api_result.header === 'undefined')
+         || api_result.header.status !== "SUCCESS" 
+         || !api_result.taxHolidays
+         || (typeof api_result.taxHolidays === 'undefined')
```

Comment:
Actually, we should be able to simplify this to something like `if (!(api_result && api_result.length))`

and then use a relevancy block to ensure required properties are defined for each item

http://docs.duckduckhack.com/frontend-reference/display-reference.html#relevancy-object-optional